src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -2266,12 +2266,9 @@ internal static uint GetValEscape(BoundExpression expr, uint scopeOfTheContainin
                     return Binder.ExternalScope;
 
                 case BoundKind.TupleLiteral:
-                    var tupleLiteral = (BoundTupleLiteral)expr;
-                    return GetTupleValEscape(tupleLiteral.Arguments, scopeOfTheContainingExpression);
-
                 case BoundKind.ConvertedTupleLiteral:
```

Comment:
>case [](start = 16, length = 4)

Not related to this line of code: consider extending `LocalRewritingValidator` to assert that no unconverted tuple literals remain after lowering. I don't think we have such a validator for initial binding.

Reply:
Emit has no support for tuple literals of any kind, so that is already checked in emit.

---
In reply to: [301753517](https://github.com/dotnet/roslyn/pull/37052#discussion_r301753517) [](ancestors = 301753517)